velox/functions/prestosql/window/tests/RankTest.cpp

```diff
@@ -46,6 +48,22 @@ TEST_F(RankTest, singlePartition) {
   });
 
   testTwoColumnInput({vectors}, "rank");
+  testTwoColumnInput({vectors}, "dense_rank");
+  testTwoColumnInput({vectors}, "percent_rank");
+}
+
+TEST_F(RankTest, singleRowPartitions) {
+  vector_size_t size = 1000;
+  auto vectors = makeRowVector({
+      makeFlatVector<int32_t>(
+          size, [](auto row) -> int32_t { return row % 1000; }),
```

Comment:
row % 1000 is the same as row given that size 1000. Let's remove % 1000 to avoid confusion.

Reply:
Also, "-> int32_t" is not necessary. Let's remove.